references/depth/stereo/presets.py

```diff
@@ -0,0 +1,111 @@
+from typing import Tuple, Union
+
+import torch
+import transforms as T
+
+
+class StereoMatchingEvalPreset(torch.nn.Module):
+    def __init__(
+        self, mean: float = 0.5, std: float = 0.5, resize_size=None, interpolation_type: str = "bilinear"
```

Comment:
Please add type hint for resize_size (I think it should be `Optional[Tuple[int, int]]` in this case)